Dockerfile

```diff
@@ -1,16 +1,27 @@
-FROM openjdk:10-slim
+ARG CANTALOUPE_VERSION=4.0.2
+FROM maven:3.6.0-jdk-11 AS MAVEN_TOOL_CHAIN
+ARG CANTALOUPE_VERSION
+ENV CANTALOUPE_VERSION=$CANTALOUPE_VERSION
+RUN mkdir -p /build && \
+    cd /build && \
+    if [ "$CANTALOUPE_VERSION" = 'latest' ] ; then curl -OL https://github.com/medusa-project/cantaloupe/archive/develop.zip; else curl -OL https://github.com/medusa-project/cantaloupe/releases/download/v$CANTALOUPE_VERSION/Cantaloupe-$CANTALOUPE_VERSION.zip; fi
+
+# unzip and compile the Cantaloupe source code if $CANTALOUPE_VERSION = 'latest'
+WORKDIR /build/
+RUN if [ "$CANTALOUPE_VERSION" = 'latest' ]; then unzip develop.zip cantaloupe-develop/* && mv cantaloupe-develop src && cd src/ && cp test.properties.sample test.properties && mvn -Pfreedeps -DskipTests clean package && cp /build/src/target/cantaloupe-?.?-SNAPSHOT.zip /build/Cantaloupe-$CANTALOUPE_VERSION.zip; fi
 
-ENV CANTALOUPE_VERSION=4.0.2
-#ENV IMAGEMAGICK_VERSION=7.0.8-14
+FROM openjdk:10-slim
+ARG CANTALOUPE_VERSION
+ENV CANTALOUPE_VERSION=$CANTALOUPE_VERSION
 
 EXPOSE 8182
 
 VOLUME /imageroot
 
 # Update packages and install tools
 RUN apt-get update -y && \
-    apt-get install -y --no-install-recommends wget unzip graphicsmagick curl imagemagick libopenjp2-tools ffmpeg python && \
-    rm -rf /var/lib/apt/lists/*
+    apt-get install -y --no-install-recommends wget unzip graphicsmagick curl imagemagick libopenjp2-tools ffmpeg python
+    #rm -rf /var/lib/apt/lists/* ### this command prevents future tinkering with apt
```

Comment:
Would be good to keep this uncommented, I think, to encourage us not to do this.

Reply:
I agree... this was a mistake, left over from when I started this work... we'll have to keep an eye out for it in the future, this probably won't be the last time we see this line commented